src/api/ledger/parse/trustline.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+const assert = require('assert');
+const utils = require('./utils');
+const flags = utils.core.Transaction.flags.TrustSet;
+
+function parseTrustline(tx) {
```

Comment:
Well there is a shortage of reviewers for this, so I appreciate the help and invite you to continue reviewing if you have the time.

It is a little tricky though because it is very much a work in progress with a lot of code that is untested an *known* to be broken. In particular, the `src/api/ledger` directory, except for the `parse` directory is mostly broken.